setup.py

```diff
@@ -7,15 +7,15 @@
 # NOTE: When updating botocore make sure to update awscli/boto3 versions below
 install_requires = [
     # pegged to also match items in `extras_require`
-    'botocore>=1.32.4,<1.33.2',
+    'botocore>=1.33.2,<1.33.14',
```

Comment:
given the various internal changes required, I think this should be `botocore>=1.33.13,<1.33.14` unless you worked through all the backwards compatibility impacts

Reply:
also note we only test one version of botocore

Reply:
The PR should be consistent in that regard. Upstream code is stable for that version range.

Reply:
ah, I see what you did, the issues is from 1.33.1->1.33.2 nice 👍🏼